### PR TITLE
fix: remove an unused variable in the kmod

### PR DESCRIPTION
**Pull request by Andreagit97**
**What type of PR is this?**

/kind bug

/kind cleanup

**Any specific area of the project related to this PR?**

/area API-version

/area driver-kmod

**Does this PR require a change in the driver versions?**

**What this PR does / why we need it**:

This unused variable causes a build failure when we use warning as errors

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by Andreagit97** [comment]
See the failure here: https://falcosecurity.github.io/libs/matrix_ARM64/#oraclelinux-515-kmod_build

File: driver/SCHEMA_VERSION [hunk 1]
```diff
@@ -1 +1 @@
-2.7.0
```
Comment (leogr) [inline driver/SCHEMA_VERSION:1]: We shouldn't bump the schema version.

https://github.com/falcosecurity/libs/blob/master/proposals/20220203-versioning-schema-amendment.md#drivers-artifacts
> - *patch* increases either when `API_VERSION`’s patch or `SCHEMA_VERSION`’s patch number has been increased **or when any other code changes have been introduced (for example, the support for a new kernel)**

This is exactly the case when `API_VERSION` is unchanged and `SCHEMA_VERSION` is unchanged, but still in the next driver release, we have to bump the driver patch number.

Does it make sense?

Comment (Andreagit97) [reply driver/SCHEMA_VERSION]: yep, makes sense thank you for pointing this out!

**Review by jasondellaluce — APPROVED** [review]
/approve

**Pull request merged**